tests/Resources/ti.ui.ios.documentviewer.test.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* global OS_VERSION_MAJOR */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+const utilities = require('./utilities/utilities');
+
+const isCI = Ti.App.Properties.getBool('isCI', false);
+
+describe.ios('Titanium.UI.iOS',  () => {
+	it('#createDocumentViewer()', () => {
+		should(Ti.UI.iOS.createDocumentViewer).be.a.Function();
+	});
+});
+
+describe.ios('Titanium.UI.iOS.DocumentViewer', () => {
+
+	let documentViewer;
+
+	beforeEach(() => {
+		documentViewer = Ti.UI.iOS.createDocumentViewer({
+			url: 'example.html'
+		});
+	});
+
+	afterEach(done => {
+		documentViewer = null;
+		done();
+	});
+
+	it('.name', () => {
+		// On macOS < 11, it is failing. https://developer.apple.com/forums/thread/125819
+		if (isCI && utilities.isMacOS() && OS_VERSION_MAJOR < 11) {
+			this.skip();
```

Comment:
There is no `this` context in an arrow function. You'll need to change the `it('.name', () => {` line above to:
``` js
it('.name', function () {
```